notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    plt.ylabel(r'Flux $\rm [e^{-1}s^{-1}]$') # should I describe LaTeX strings here?
</pre><p>You should at least mention that you're using LaTeX strings for "nice formatting" or something to that effect</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>